README.md

```diff
@@ -1,3 +1,5 @@
+yo
```

Comment:
actually it says yo and not hi as the commit message suggests

Reply:
sorry about that. will fix